packages/mobile/locales/en-US/global.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
Is this file here by mistake?  Most of these strings don't have anything to do with Persona